python_bindings/correctness/the_sort_function.c

```diff
@@ -3,72 +3,24 @@
  * gcc -std=c99 the_sort_function.c -shared -o the_sort_function.so
  */
 
+#include "HalideRuntime"
```

Comment:
Does this work without the .h?

Reply:
No, but apparently it's not currently actively built and tested :-/